src/test/java/com/google/cloud/genomics/dataflow/pipelines/CountReadsITCase.java

```diff
@@ -36,6 +35,8 @@
   final String TEST_STAGING_GCS_FOLDER = System.getenv("TEST_STAGING_GCS_FOLDER");
   // this file shouldn't move.
```

Comment:
tiny nit: Comments should be sentences starting with capital letters here and elsewhere.